src/main/java/seedu/address/logic/parser/FilterCommandParser.java

```diff
@@ -148,11 +148,22 @@ private List<Predicate<Client>> collectPredicates(ArgumentMultimap argMultimap)
         if (argMultimap.getValue(PREFIX_TIER).isPresent()) {
             String substring = parseFieldForFilterCommand(() -> parseField(() -> argMultimap.getValue(
                     PREFIX_TIER).get(), errors));
+            if (substring.isEmpty()) {
+                errors.add("Tier has not been provided any value to filter by.\n"
+                        + "Please specify a status label to filter by, such as 'Gold', 'Silver', 'Bronze', 'Reject', "
+                        + "or 'NA'\n"
+                        + "To filter for clients without a visible tier label beside their name: filter s/ NA");
+            }
```

Comment:
I think its best to extract this out and put this as a string at the top instead.

Reply:
Good point, let me implement this